streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryLRUCacheStoreTest.java

```diff
@@ -93,4 +116,16 @@ public void testEvict() {
         assertTrue(driver.flushedEntryRemoved(3));
         assertEquals(3, driver.numFlushedEntryRemoved());
     }
+
+
+    private List<KeyValue<Integer, String>> fillKeyValueList(List<Integer> keys, List<String> values) {
```

Comment:
I don't think we need this. You can just inline the creation of the `List<KeyValue>` see above.